src/picongpu/include/traits/attribute/GetCharge.hpp

```diff
@@ -35,25 +36,28 @@ namespace detail
 
 /** Calculate the real charge of a particle
  *
- * use attribute `chargeState` to calculate the charge
+ * use attribute `boundElectrons` and the proton number from 
+ * flag `atomicNumbers` to calculate the charge
  */
-template<bool T_HasChargeState>
-struct LoadChargeState
+template<bool T_HasBoundElectrons>
+struct LoadBoundElectrons
 {
 
     template<typename T_Particle>
```

Comment:
pls add a doxygen string for the params here, I am already confused what "partialResult" means :)